x-pack/plugins/observability_solution/dataset_quality/README.md

```diff
@@ -18,7 +18,31 @@ You can also run a specific test by passing the filepath as an argument, e.g.:
 yarn jest --config x-pack/plugins/observability_solution/dataset_quality/jest.config.js x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_streams/get_data_streams.test.ts
 ```
 
-#### API integration tests
+### Agnostic-deployement API tests
+
+The agnostic-deployement API tests are located in [`x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality`](/x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/).
```

Comment:
>The agnostic-deployement API tests

The deployment-agnostic API tests

Reply:
addressed in [b8963e2](https://github.com/elastic/kibana/pull/193144/commits/b8963e2b71f08da4c73f5d946566fd171e8e0846).